OpenRA.Mods.Common/Traits/Transforms.cs

```diff
@@ -103,7 +103,7 @@ public void DeployTransform(bool queued)
 				// Only play the "Cannot deploy here" audio
 				// for non-queued orders
 				foreach (var s in info.NoTransformSounds)
-					Game.Sound.PlayToPlayer(self.Owner, s);
+					Game.Sound.PlayToPlayer(SoundType.World, self.Owner, s);
```

Comment:
Notifications on UI and sounds for world always seemed like the intention to me, so I fail to see the giant mess. There are some cases where they mixture, like at crates though, but again those looks like legacy edge-cases to me..